packages/avatars/src/styled/StyledAvatar.ts

```diff
@@ -5,129 +5,59 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { css, keyframes, ThemeProps, DefaultTheme } from 'styled-components';
+import styled, { css, ThemeProps, keyframes, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import { math } from 'polished';
-import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+
+import { IAvatarProps, SIZE } from '../types';
 import { StyledText } from './StyledText';
-import { IAvatarProps } from '../types';
+import { StyledStatusIndicator } from './StyledStatusIndicator';
+import { getStatusColor } from './utility';
 
 const COMPONENT_ID = 'avatars.avatar';
 
 const TRANSITION_DURATION = 0.25;
 
 const badgeStyles = (props: IStyledAvatarProps & ThemeProps<DefaultTheme>) => {
-  let content = `''`;
-  let position = '0';
-  let padding = '0';
-  let minWidth = '0';
-  let height = '0';
-  let fontSize = '0';
-  let borderWidth = props.theme.shadowWidths.sm;
-
-  if (props.status === 'active') {
-    position = `${props.theme.space.base * -1}px`;
-
-    if (props.size === 'small') {
-      fontSize = props.theme.fontSizes.xs;
-      height = `${props.theme.space.base * 4}px`;
-      minWidth = fontSize;
-      padding = `${props.theme.space.base - 1}px`;
-      content = 'attr(data-badge)';
-    } else if (props.size === 'extrasmall') {
-      height = `${props.theme.space.base * 2}px`;
-      minWidth = height;
-    } else if (props.size === 'extraextrasmall') {
-      position = math(`${position} + 1`);
-      height = `${props.theme.space.base + 1}px`;
-      minWidth = height;
-      borderWidth = math(`${borderWidth} - 1`);
-    } else {
-      fontSize = props.theme.fontSizes.xs;
-      height = `${props.theme.space.base * 5}px`;
-      minWidth = fontSize;
-      padding = `${props.theme.space.base + 1}px`;
-      content = 'attr(data-badge)';
-    }
-  } else if (props.status === 'available') {
-    position = `${props.theme.space.base * -1}px`;
-
-    if (props.size === 'large') {
-      height = `${props.theme.space.base * 3.5}px`;
-    } else if (props.size === 'small') {
-      height = `${props.theme.space.base * 2.5}px`;
-    } else if (props.size === 'extrasmall') {
-      height = `${props.theme.space.base * 2}px`;
-    } else if (props.size === 'extraextrasmall') {
-      position = math(`${position} + 1`);
-      height = `${props.theme.space.base + 1}px`;
-      borderWidth = math(`${borderWidth} - 1`);
-    } else {
-      height = `${props.theme.space.base * 3}px`;
-    }
-
-    minWidth = height;
-  } else if (props.size === 'large') {
-    position = math(`${props.theme.shadowWidths.sm} + 1`);
-  } else if (props.size === 'small') {
-    position = math(`${props.theme.shadowWidths.sm} - 1`);
-  } else if (props.size === 'medium') {
-    position = props.theme.shadowWidths.sm;
+  const [xxs, xs, s, m, l] = SIZE;
```

Comment:
Is the smaller default badge size change intentional? The original design intent was for the `active` notification change to result in a more pronounced "bounce" effect in order to get the user's attention. The current animation does not follow original intent and is very subtle.

For comparison, visit https://zendeskgarden.github.io/react-components/?path=/story/packages-avatars-avatar--avatar&args=status:available and then set a `badge` control – observing the effect.

Reply:
Good call on that, I reverted to the original behavior so it should be back now.

Reply:
The animation is restored. But I'm 99% sure the badge resize from the original 20px down to 16px is a result of a bug in Figma where the badge was not originally sized correctly.